app/controllers/form_templates_controller.rb

```diff
@@ -23,4 +26,39 @@ def show
   def ensure_form_templates_enabled
     raise Discourse::InvalidAccess.new unless SiteSetting.experimental_form_templates
   end
+
+  def process_template(template_content)
```

Comment:
this is not tested @jdmartinez1062

Reply:
also a good followup would be to use services for this

Reply:
added test now